src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -232,6 +233,20 @@ private DocumentNode GenerateSqlGraphQLObjects(RuntimeEntities entities, Diction
                 objectTypes[entityName] = QueryBuilder.AddQueryArgumentsForRelationships(node, inputObjects);
             }
 
+            Dictionary<string, FieldDefinitionNode> fields = new();
+            NameNode nameNode = new(value: GraphQLUtils.DB_OPERATION_RESULT_TYPE);
+            FieldDefinitionNode field = MutationBuilder.GetDefaultResultFieldForMutation();
+
+            fields.TryAdd("result", field);
+
+            objectTypes.Add(GraphQLUtils.DB_OPERATION_RESULT_TYPE, new ObjectTypeDefinitionNode(
+                location: null,
+                name: nameNode,
+                description: null,
+                new List<DirectiveNode>(),
+                new List<NamedTypeNode>(),
+                fields.Values.ToImmutableList()));
+
```

Comment:
this object is added even when DWSQL is not used as a datasource? Would this also be created if DWSQL only defined queries in its GraphQL schema? 

I'm also on the fence about this, but `GetDefaultResultFieldForMutation()` may be better placed as a private function in this class rather than the mutation builder class. While it is relevant to mutations, it is really just building a GraphQL Object type definition, that happens to be used for mutations, and not building a mutation explicitly.

Reply:
I can make it private within the same class. The reason that I did not add DW specific logic here to create this only when its DW was for two reasons:
1. This could be used by other datasources in the future. 
2. as per my understanding, adding this to the schema (even if its not used) should not have any perf impact or any impact on the schema. Hence, did not want to add any db specific logic here. 

Let me know if you think otherwise.

Reply:
Makses sense. created GetDbOperationResultField.